build.sh

```diff
@@ -335,12 +335,13 @@ download_sync() {
     TMP_DIR="$TMPDIR/sync_bin"
     mkdir -p "${TMP_DIR}"
     SYNC_TMP_TAR="${TMP_DIR}/sync-${REALM_SYNC_VERSION}.tar.xz.tmp"
+    rm -rf "$SYNC_TMP_TAR" # ensure we don't have a corrupt download
```

Comment:
not much point in caching the download if we just delete the tarball on every run...

if this is to handle corrupt downloads, it should be done as error handling rather than preemptively.